sdk/inc/azure/core/internal/az_log_internal.h

```diff
@@ -11,22 +11,19 @@
 
 #include <azure/core/_az_cfg_prefix.h>
 
-// If the user hasn't registered any classifications, then we log everything.
+#ifdef AZ_NO_LOGGING
```

Comment:
Please revert to ifndef, I have more detail in the main PR. Especially here, this is supposed to be a minimalistic change.